src/components/Button/Button.tsx

```diff
@@ -1,38 +1,45 @@
 import React from 'react';
 
-interface MenuButtonProps {
-  text: string;
+interface ButtonProps {
   type?: 'info' | 'cancel' | 'confirm';
```

Comment:
doesnt need to happen but type might be a bad name for this since that attribute already exists on button and what if we wanted to be able to pass in a type like 'submit'

Reply:
no u